CIP-?/README.md

```diff
@@ -0,0 +1,109 @@
+---
+CIP: ?
+Title: Cardano URI Scheme Extensions
```

Comment:
```suggestion
Title: URI Scheme - Block and transaction objects
```
1 - Since https://github.com/cardano-foundation/CIPs/pull/559 we have expected more CIPs to define new CIP-0013 `authority` keywords, and since https://github.com/cardano-foundation/CIPs/pull/594 we've been grouping e.g. the wallet API extensions with the CIP `Title:` prefix `Web-Wallet Bridge -`.

Therefore I would suggest we begin CIPs for new CIP-0013 extensions with this title prefix: `URI Scheme -`

2 - I also think "common objects" is not specific enough to indicate what these links are referring to (the term "object" is too broad) and whenever possible they should be mentioned by function (or in some cases the `authority` itself).

@Quantumplation I'm setting the PR title as such but we can follow whatever is agreed upon here.  @Crypto2099 @Ryun1